py3status/modules/bluetooth.py

```diff
@@ -28,11 +28,6 @@
 """
 
 import re
-import shlex
-
-from subprocess import check_output
-
-BTMAC_RE = re.compile(r'[0-9A-F:]{17}')
```

Comment:
Is there a reason you removed this?  I see no point in compiling the re every time.

also `macs = set(re.findall(BTMAC_RE, out))` can just be `macs = set(BTMAC_RE.findall(out))`